.github/workflows/main.yml

```
@@ -82,3 +82,11 @@ jobs:
       - name: Mark the job as unsuccessful
         if: contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')
         run: exit 1
+  fmt:
```

No need to abbreviate here:

```suggestion
  format:
```

---

Removed abbreviation, now using `Format:` (with a capital F) to match the Build step's capitalization